### PR TITLE
Add border to embedded tinder chat

### PR DESCRIPTION
**Pull request by jack829**
RNR-21

### CODE REVIEW

File: src/screens/Chat.js [hunk 1]
```diff
@@ -16,15 +16,16 @@ import {
...
 import AuthInterface from '../AuthInterface'
 import SocketInterface from '../SocketInterface'
-import ChatMessage from '../components/Chat/ChatMessage';
-import { splitSuggestions } from '../helpers/messageParser';
-import { SUGGESTION_STATUSES } from '../Constants';
-import DummyWrapper from '../HOC/DummyWrapper';
+import ChatMessage from '../components/Chat/ChatMessage'
+import { splitSuggestions } from '../helpers/messageParser'
+import { SUGGESTION_STATUSES } from '../Constants'
+import TinderChatEmbedded from './TinderChat/TinderChatEmbedded';
```
Comment (dblanchard13) [inline src/screens/Chat.js]: NIT can you axe the semi-colon?

File: src/screens/TinderChat/TinderChat.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import API from '../../API'
 import AuthInterface from '../../AuthInterface'
 import SocketInterface from '../../SocketInterface'
 import ChatMessage from '../../components/Chat/ChatMessage';
+import { COLOR_TINDER_FADED } from '../../globals/styles';
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChat.js]: NIT can you axe the semi-colons on the above 2 lines?

File: src/screens/TinderChat/TinderChat.js [hunk 3]
```diff
@@ -198,20 +199,46 @@ class Chat extends Component {
     )
   }
 
-  render() {
-    if ( !this.props.conversation.TinderMessages ) return <Loading />
+  getKeyboard() {
+    if (this.props.isEmbedded) return;
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChat.js]: NIT can you axe the semi-colon?

File: src/screens/TinderChat/TinderChat.js [hunk 4]
```diff
@@ -198,20 +199,46 @@ class Chat extends Component {
...
+      return <Loading embed={ this.props.isEmbedded ? 'tinder' : null }/>
+    }
     const { messages, messageDraft } = this.state
-    const data = messages.length ? messages : [{ id: 'dummy_id', noMessages: true }]
+    let data = [{ id: 'dummy_id', noMessages: true }]
 
-    return (
-      <View style={ styles.container }>
+    if (messages.length) {
+      data = messages.slice().reverse();
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChat.js]: NIT can you axe the semi-colon?

File: src/screens/TinderChat/TinderChatEmbedded.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+          <View style={ collapseToggleContainer }>
+            <TouchableOpacity onPress={this.onToggleExpand.bind(this)}>
+              <Icon name='ios-arrow-up' color="#fff" size={20}/>
+            </TouchableOpacity>
+          </View>
+        </View>
+      )
+    }
+
+    return currentView;
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChatEmbedded.js]: NIT can you axe the semi-colon?

File: src/screens/TinderChat/TinderChatEmbedded.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    borderColor: '#fff',
+    borderRadius: 3
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChatEmbedded.js]: NIT can you add a hanging comma?

File: src/screens/TinderChat/TinderChatEmbedded.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    flex: 1,
+  },
+  expandToggleContainer: {
+    paddingTop: 4,
+    paddingRight: 2,
+  },
+  collapseToggleContainer: {
+    backgroundColor: COLOR_TINDER_FADED,
+    flexDirection: 'row',
+    justifyContent: 'flex-end'
```
Comment (dblanchard13) [inline src/screens/TinderChat/TinderChatEmbedded.js]: NIT can you add a hanging comma?

File: src/components/Loading.js [hunk 8]
```diff
@@ -5,6 +5,7 @@ import {
   StatusBar
 } from 'react-native'
 import AppText from './AppText'
+import { COLOR_TINDER_FADED } from '../globals/styles';
```
[the remark this answers is not in the document]
Comment (jack829) [reply src/components/Loading.js]: Oh yeah. AutoImport adds them automatically and I can't figure out how to turn that off

**Comment by jack829** [comment]
done

**Pull request merged**